cmake/core-files.cmake

```diff
@@ -444,13 +444,16 @@ set(MBGL_CORE_FILES
     include/mbgl/util/geo.hpp
     include/mbgl/util/geojson.hpp
     include/mbgl/util/geometry.hpp
+    include/mbgl/util/ignore.hpp
     include/mbgl/util/image.hpp
+    include/mbgl/util/interpolate.hpp
     include/mbgl/util/logging.hpp
     include/mbgl/util/noncopyable.hpp
     include/mbgl/util/optional.hpp
     include/mbgl/util/platform.hpp
     include/mbgl/util/projection.hpp
     include/mbgl/util/range.hpp
+    include/mbgl/util/rapidjson.hpp
```

Comment:
We've kept this private because it requires having the RapidJSON headers present. What's the reason for making this public?